extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/app.js

```diff
@@ -105,11 +105,15 @@ angular.module('PaperUI', [ 'PaperUI.controllers', 'PaperUI.controllers.control'
             redirectTo : '/control'
         });
     }
+    $mdDateLocaleProvider.shortMonths = [ 'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec' ];
```

Comment:
Wouldn't it be a little bit better to move that to the else branch instead of assign it first with the english once also if the paperui.language is de? Or do we need to declare it outside of the if scope first?